packages/api/src/controllers/stream.test.js

```diff
@@ -1004,6 +1004,7 @@ describe("controllers/stream", () => {
           expect(res.status).toBe(204);
 
           await hookSem.wait(1000);
+          delete hookPayload.time;
           expect(hookPayload).toEqual({
             id: expect.stringMatching(uuidRegex),
             event: "stream.detection",
```

Comment:
**nit:** I think this could work too:
```suggestion
          expect(hookPayload).toEqual({
            id: expect.stringMatching(uuidRegex),
            event: "stream.detection",
            time: expect.any(Number),
```
(horrible diff from github, I only stopped deleting the `time` field and started checking it with `expect.any(Number)`)